### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Job View_High Contrast: High Contrast Black & White themes are not adapted by the Table

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3598]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #MAS4.3.1; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 4.3.1

**Note:
Entire job view section is not adopting the high contrast settings**

**Repro Steps:**
1)Turn on High Contrast Black & White
2)Launch IntelliJ IDEA
3)Tab till Create New Project button and press enter. Navigate to Azure Spark/HDInsight-->Spark Project Scala and press Next button and fill all the details to create a new project
4)Tab till Azure Spark/HDInsight-->Spark Project Scala
5)Tab till Build Tools-->Maven
6)Tab till Next button and press enter
7)Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
8)Tab till HDInsight and link a cluster
9)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
10)Navigate to New-->Scala Class and press enter
11)Tab till object from menu and select it using keyboard
12)Override the code and Navigate to project name-->src-->main-->scala-->Project Name and right click on it & Tab till Submit Spark Application and press enter
13)Fill all the required details & Tab till finish button and press enter
14)Navigate to jobs under the cluster (which we have used) and press enter
15)Verify whether the table is adapting the high contrast black & white themes or not

**Actual Result:**
The table section is not adapting the high contrast black & white themes 

**Expected Result:**
The table section should adapt the high contrast black & white themes

**User Impact:**
Users with color blindness will find difficulty in viewing the table, if it doesn't adapt the high contrast black & white themes

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Attachment:**
![Bug_3598_not adopting high contrast settings](https://user-images.githubusercontent.com/42638450/65421680-f2c82200-de21-11e9-85f1-0525f220ef36.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749701
